package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "casbin-express-authz",
-  "version": "2.0.0",
+  "version": "3.0.0",
   "description": "casbin-express-authz is an authorization middleware for Express",
-  "main": "authz.js",
+  "main": "build/authz.js",
   "scripts": {
-    "prepublish": "npm run lint && npm run test",
-    "lint": "eslint \"authz.js\"",
+    "prepublish": "npm run build && npm run lint && npm run test",
```

Comment:
I thinks should run `lint`, `test`, `build` in turn.

Reply:
I think I have to run ``build`` before ``test`` as test will not run if the ``.ts`` files are not transpiled to ``.js`` files.

Reply:
Use `ts-jest` is working.

Reply:
Added